diff --git a/dist/module.mjs b/dist/module.mjs
index 52db9f5cc77a93ca93b2c7b743a8f1cb19dd1e12..221c708314728b12498182b3db20d6d3a1ce979b 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -122,7 +122,7 @@ const module = defineNuxtModule({
     nuxt.options.alias["#apollo"] = resolve(nuxt.options.buildDir, "apollo");
     addPlugin(resolve("runtime/plugin"));
     addImports([
-      { name: "gql", from: "graphql-tag" },
+      // { name: "gql", from: "graphql-tag" },
       ...[
         "useApollo",
         "useAsyncQuery",
